datafusion/sqllogictest/test_files/array.slt

```diff
@@ -4296,10 +4384,10 @@ select
   array_ndims(column4)
 from array_ndims_table;
 ----
-0 1 2 5
-0 1 2 5
-0 1 2 5
-0 1 2 5
+1 1 2 5
+1 1 2 5
+NULL 1 2 5
```

Comment:
It should be an error here too

Reply:
**Schema (PostgreSQL v15)**

    

---

**Query #1**

    select array_ndims(ARRAY[null]);

| array_ndims |
| ----------- |
| 1           |

---

Reply:
I filed #9257 and #9258 for these bugs.

Reply:
You are right, I thought it should be an error just like array_ndims(NULL) is